lib/watir/alert.rb

```diff
@@ -85,14 +85,6 @@ def exists?
     end
     alias present? exists?
 
-    #
-    # @api private
-    #
-
-    def selector_string
```

Comment:
We should wait until `#when_present` is deprecated before removing this. If anyone is doing `browser.alert.when_present.set('1')`, they will get an "undefined local variable or method `selector_string'" error.

Reply:
aha! I couldn't figure out where that was being used. Thanks.